lib/ui/views/ib/syntaxes/ib_md_tag_syntax.dart

```diff
@@ -22,7 +22,7 @@ class IbMdTagSyntax extends md.BlockSyntax {
       return md.Element.text('h5', text);
     }
 
-    // (TODO) Fix Pop Quizes
+    // Pop Quizzes

```

Comment:
I assume the Pop Quizzes worked fine already, but because changing the comment is such a small change, you added this in this PR only, right?

Reply:
Pop Quizzes working fine but somehow I forgot to remove TODO and while checking code, I found this outdated comment and added.